www/templates/control/main-control.html

```diff
@@ -10,45 +10,68 @@
       </div>
   </div> -->
 
-  <ion-content style="background-color: #262A30;">
+  <ion-content>
 <!--     <div style="text-align: center;">
       <img src="img/avatar_1.png" ng-style="getAvatarStyle()">
     </div> -->
     
-    <div style="background-color: white; height: 50px;">
-      <div style="padding: 15px 15px; float: left;">{{settings.auth.email}}</div>
+    <div class="control-list-item">
```

Comment:
Awesome to allow choice between the two. I think that doing that will encourage you to make the settings configurable rather than hard-coded.